tools/clang/test/CodeGenDXIL/hlsl/wavesize/wavesize-range-not-int.ll

```diff
@@ -0,0 +1,42 @@
+; XFAIL:*
```

Comment:
A comment explaining why this is expected to fail and referencing a bug would be helpful